src/schnetpack/properties.py

```diff
@@ -44,7 +42,15 @@
 forces: Final[str] = "forces"
 stress: Final[str] = "stress"
 
-total_charge: Final[str] = "total_charge"
+
 partial_charges: Final[str] = "partial_charges"
 dipole_moment: Final[str] = "dipole_moment"
-polarizability: Final[str] = "polarizability"
+polarizability: Final[str] = "polarizability"  
+total_charge: Final[str] = "total_charge"
+
+    
+electrostatic: Final[str] = "electrostatic"
+zbl: Final[str] = "zbl"
+dispersion: Final[str] = "dispersion"
+physnet_aggregate: Final[str] = "physnet_aggregate"
```

Comment:
this should not be a constant here. you can internally call the atomwise energy `short_range` and then the aggregated energy just energy, or energy_U0 for QM9 (${property})